src/qt/optionsmodel.cpp

```diff
@@ -212,6 +214,9 @@ bool OptionsModel::setData(const QModelIndex & index, const QVariant & value, in
             emit coinControlFeaturesChanged(fCoinControlFeatures);
             }
             break;
+        case DisableUpdateCheck:
+            fDisableUpdateCheck = value.toBool();
+            break;
```

Comment:
I think we need to add:
```cpp
settings.setValue("fDisableUpdateCheck", fDisableUpdateCheck);
```
...here to persist the checkbox, and:
```cpp
if (settings.contains("fDisableUpdateCheck"))
    SoftSetBoolArg("-disableupdatecheck", settings.value("fDisableUpdateCheck").toBool());
```
...to `OptionsModel::Init()`.